nevergrad/optimization/optimizerlib.py

```diff
@@ -254,16 +257,11 @@ def _internal_tell(self, x: tp.ArrayLike, loss: tp.FloatLoss) -> None:
             self._doerr_index = -1
         if self.mutation == "doerr":
             self._doerr_current_best = min(self._doerr_current_best, loss)
-        self._sigma *= 2.0 if loss <= self.current_bests["pessimistic"].mean else 0.84
         if self.mutation == "adaptive":
-            factor = (
-                1.2 if loss <= self.current_bests["pessimistic"].mean else 0.731
-            )  # 0.731 = 1.2**(-np.exp(1)-1)
+            factor = 1.2 if loss <= previous_loss else 0.731  # 0.731 = 1.2**(-np.exp(1)-1)
```

Comment:
@teytaud we should probably do the same thing here? = factor=1 if the loss is not modified?